app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
I think your analysis is correct. I think this is the way it is because we overlooked the fact that this should be an AR validation rather than controller logic. What we have now (https://github.com/18F/dolores-landingham-bot/blob/5c5b06043a5b08d139fc5d08560c2d6a0f26f894/app/controllers/employees_controller.rb#L7) is def non-ideal and I think this branch would be a great opp to improve upon it!

No worries on timing, I know how it can be finding time to work on side stuff! :)